core/src/main/scala/com/lightbend/paradox/markdown/Directive.scala

```diff
@@ -311,7 +311,8 @@ case class SnipDirective(page: Page, variables: Map[String, String])
         } else new File(page.file.getParentFile, source)
       val text = Snippet(file, labels)
       val lang = Option(node.attributes.value("type")).getOrElse(Snippet.language(file))
-      new VerbatimNode(text, lang).accept(visitor)
+      val group = Option(node.attributes.value("group")).getOrElse("")
```

Comment:
The tab name doesn't seem too easy to reach at this point, but defaulting to the `lang` seems reasonable, too - applied to #107